pkg/tcpip/stack/ndp.go

```diff
@@ -720,6 +743,83 @@ func (ndp *ndpState) handleRA(ip tcpip.Address, ra header.NDPRouterAdvert) {
 	}
 }
 
+func (ndp *ndpState) sendRouterAdvert() {
+	ref := ndp.nic.getRefOrCreateTemp(header.IPv6ProtocolNumber, header.IPv6AllRoutersMulticastAddress, NeverPrimaryEndpoint, forceSpoofing)
+	la := tcpip.Address(net.ParseIP("fe80::2")) //
+	r := makeRoute(header.IPv6ProtocolNumber, la, header.IPv6AllNodesMulticastAddress, ndp.nic.linkEP.LinkAddress(), ref, false, false)
+	defer r.Release()
+
+	// Route should resolve immediately since
+	// header.IPv6AllRoutersMulticastAddress is a multicast address so a
```

Comment:
the resolution would occur on `header.IPv6AllNodesMulticastAddress` since that is the destination of the IP packet.